docs/configure.rst

```diff
@@ -0,0 +1,163 @@
+=====================
+Install the Croud CLI
+=====================
+
+This page shows you how to install the `Croud CLI`_. The Croud command-line
+interface (CLI) lets you issue commands to create and manage CrateDB Cloud
+organizations, projects, and resources, and deploy CrateDB clusters.
+
+In this tutorial, the *Croud CLI tool* is reffered to as *Croud*.
+
+.. NOTE::
+
+   Croud requires `Python 3.6 or higher`_.
+
+.. SEEALSO::
+
+   For an outline of available Croud commands, take a look at the
+   `Croud CLI section`_.
+
+Install
+=======
+
+To install Croud, start your system's command-line interface and run this
+command:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+Next, verify if the installation was successful by running:
+
+.. code-block:: console
+
+    sh$ croud -v
+
+If Croud was successfully installed, the command outputs the Croud version
+number, like so:
+
+.. code-block:: console
+
+   croud 0.16.0.
+
+Update Croud
```

Comment:
per above comment, remove this subsection title and merge all this Croud content into one section